.github/workflows/ci-worker.yaml

```diff
@@ -6,10 +6,10 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-      - name: Set up Go 1.18
+      - name: Set up Go 1.19
         uses: actions/setup-go@v3
         with:
-          go-version: 1.18.5
+          go-version: 1.19
```

Comment:
are there other places we should update our declared Go version as well? `go.mod` file or other CI?

Reply:
It is updated in `go.mod` .. not sure if we use any other CI platforms